lib/flume/api.ex

```diff
@@ -70,6 +70,11 @@ defmodule Flume.API do
       end
 
       def worker_context, do: Pipeline.Context.get()
+
+      @doc """
+      Returns count of jobs in the pipeline which are yet to be processed.
+      """
+      defdelegate job_counts(queues), to: Manager
```

Comment:
> @HarshBalyan Can you add these details to the doc section?
> 
>     * Example usage
> 
>     * [Documentation metadata](https://hexdocs.pm/elixir/writing-documentation.html#documentation-metadata) `@doc since: "0.2.0"`
> 
>     * [Function specification](https://hexdocs.pm/elixir/typespecs.html)

- Example usage ✔️ 
- Function specification ✔️ 
- Documentation metadata isn't available in elixir 1.6.6

Reply:
> Are we expecting the user to provide pipeline-name to this function?
> If yes, the argument name should be `pipelines`?

We are expecting a list of `Flume.Pipeline.queue`. Therefore, I decided to call the argument `queues`.

Reply:
👍